react/wbooks/src/app/components/Login/index.js

```diff
@@ -17,8 +19,10 @@ class Login extends Component {
   handleLogin = async event => {
     event.preventDefault();
     try {
-      const response = await login(this.state); // eslint-disable-line no-unused-vars
+      const response = await login(this.state);
       this.setState({ isError: false, errorMessages: [] });
+      set(ACCESS_TOKEN, response.access_token);
+      this.props.history.push(Routes.HOME);
```

Comment:
use destructuring

Reply:
where do you mean to apply destructuring? inside the function? I no see difference in doing that since `history` in the only property I use of this.props

Reply:
its like a readability standard, because is not an arrow function with instant return.